website/docs/docs/core/connect-data-platform/postgres-setup.md

```diff
@@ -93,3 +93,57 @@ If the database closes its connection while dbt is waiting for data, you may see
 
 If `dbt-postgres` encounters an operational error or timeout when opening a new connection, it will retry up to the number of times configured by `retries`. The default value is 1 retry. If set to 2+ retries, dbt will wait 1 second before retrying. If set to 0, dbt will not retry at all.
 
+
+### `psycopg2` vs `psycopg2-binary`
+
+`psycopg2-binary` is installed by default when installing `dbt-postgres`.
+This is ideal for development and testing workflows where performance is less of a concern.
+However, production environments will benefit from `psycopg2`, which is built from source for that particular operating system and archtecture.
+
+<VersionBlock firstVersion="1.8">
+
+To use `psycopg2`, simply uninstall `psycopg2-binary` and install the equivalent version of `psycopg2` after installing `dbt-postgres`:
+```bash
+pip install dbt-postgres
+if [[ $(pip show psycopg2-binary) ]]; then
+    PSYCOPG2_VERSION=$(pip show psycopg2-binary | grep Version | cut -d " " -f 2)
+    pip uninstall -y psycopg2-binary
```

Comment:
Maybe: 

`pip uninstall -y psycopg2-binary &&  pip install psycopg2==$PSYCOPG2_VERSION` would be nice because then we chain those commands together rather than trying to install on an unsuccessful uninstall.